Docker-compose.yml

```
@@ -0,0 +1,27 @@
+version: '3'
+services: 
+  app:
+    build: .
+    ports:
+      - "5000:5000"
+    command: bundle exec rails s -p 5000
+    depends_on:
+      - iis-sandbox
+      - v2-to-fhir
+    links:
+      - iis-sandbox
+      - v2-to-fhir
+    environment:
+      IIS_SANDBOX_USERNAME: mitre
+      IIS_SANDBOX_PASSWORD: mitre
+      IIS_SANDBOX_FACILITY_ID: MITRE Healthcare
+      IIS_SANDBOX_HOST: http://localhost:8081
+      V2_TO_FHIR_HOST: http://localhost:3000
+  iis-sandbox:
+    build: ./IIS-Sandbox
+    ports:
+      - "8081:8080"
+  v2-to-fhir: 
+    build: ./hl7v2-fhir-vaccine-credential-converter
+    ports:
+      - "3000:3000"
```

```suggestion
    expose:
      - "3000"
```

We can publish this port without exposing it on the host machine ([Docker-compose#expose](https://docs.docker.com/compose/compose-file/compose-file-v3/#expose))